.github/workflows/build.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       run: |
         export PHP_CS_FIXER_IGNORE_ENV=1
         #vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php -v --dry-run --diff --using-cache=no
-        vendor/bin/phpunit --debug --verbose
```

Comment:
Why's this removed?

Reply:
Because --verbose doesn't exist anymore and I think the output was better without --debug